### PR TITLE
Update README.md file

### LINKED ISSUE #1: Add setup description in readme
Currently there are no instructions for the setup of this project in the README.md file.
Such description should be added.

### PR DESCRIPTION
**Pull request by dim4o**
resolves #1

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,26 @@
-# atmosphere-integration-tests
\ No newline at end of file
+# atmosphere-integration-tests
+This is the integration tests of the project
```
Comment (dakatapetrov) [inline README.md]: These are the ... of the ATMOSPHERE project

File: README.md [hunk 2]
```diff
@@ -1 +1,26 @@
...
+
+> Make sure you publish these projects to your local Maven repository
+
+### Chrome driver setup
+* download Chrome Web Driver from [here](https://sites.google.com/a/chromium.org/chromedriver/) and unzip.
+* open `agent.properties` from `atmosphere-integration-tests` root directory and find `chromedriver.executable.path` property. Change the value to point to your chromedriver executable file.
+
+### Build the projects
+Make sure you have connected physical device and emulator.
+You can build the project using the included gradle wrapper by running `./gradlew build` on Linux/macOS
```
Comment (dakatapetrov) [inline README.md]: Probably it's a good idea to separate the commands on different lines so they are more visible.

Also we can add some info about the maven publications.

File: README.md (hunk 1 above)
Comment (dim4o) [inline README.md]: Updated

File: README.md (hunk 2 above)
Comment (dim4o) [inline README.md]: Updated - now the commands are separated on different lines.

Comment (dim4o) [inline README.md]: I cannot understand the last part of the second comment - there aren't any Maven publications for 'atmosphere-integration-tests` project.

**Pull request merged**